src/sentry/migrations/0001_initial.py

```diff
@@ -489,15 +489,15 @@ class Migration(migrations.Migration):
                                                                                       (82,
                                                                                        b'rule.remove'),
                                                                                       (100,
-                                                                                       b'serivcehook.create'),
+                                                                                       b'servicehook.create'),
                                                                                       (101,
-                                                                                       b'serivcehook.edit'),
+                                                                                       b'servicehook.edit'),
                                                                                       (102,
-                                                                                       b'serivcehook.remove'),
+                                                                                       b'servicehook.remove'),
                                                                                       (103,
-                                                                                       b'serivcehook.enable'),
+                                                                                       b'servicehook.enable'),
                                                                                       (104,
-                                                                                       b'serivcehook.disable'),
+                                                                                       b'servicehook.disable'),
```

Comment:
Don't change migrations, let's leave as-is. You'll probably need to generate a new migration, since I guess this was fixed elsewhere on the model.

Reply:
fwiw it should be safe to change this if its changed everywhere

Reply:
Yeah it won't cause any db operations, but I just generally think we shouldn't edit migrations

Reply:
imo if its correcting to be the actual word used everywhere else its fine.. its not like the code is immutable. this would ensure that if a future migration operated on it its got the right fields and you dont gotta intentionally spell things wrong

likely doesn't matter, but same argument about "might as well just let code formatters run, theres not really any harm other than git history"

Reply:
Yeah, I didn't know the best practice here, so was waiting for review.

Reply:
I'm not against fixing this in the model, I'd just prefer to fix the model and generate a new migration to store the updated state. The other thing here is that I don't see the model in this pr, since I guess it got split up?

Reply:
but y tho? more migrations = more slow.. if its just fixing a literal in-code-only thing id just fix it in all src code and call it a day. done this numerous times in other repos never caused a problem.

agree tho if its not fixed in the model thats a whole diff issue

Reply:
It's more of a procedure thing - It's much easier to tell engineering as a whole "pls don't modify migrations". As we get more and more people working on the codebase I'd prefer to just have simple rules when possible, since most people don't have a deep understanding of how django migrations works.

I don't really mind in this case if you're very pro modifying the migration, but in general I'd just tell people to make a new migration. We've generated 94 migrations in under a year, so if they're really making things slower then we're going to have to collapse them at some point anyway.

Reply:
dont care that much - was just trying to say we shouldnt have a hard and fast rule "you cant change migrations"